etna/models/nn/tft.py

```diff
@@ -26,7 +26,7 @@
     from pytorch_lightning import LightningModule
 
 
-class TFTModel(Model, PredictIntervalAbstractModel, _DeepCopyMixin):
+class TFTModel(Model, MultiSegmentPredictionIntervalModel, _DeepCopyMixin):
```

Comment:
We don't need Model here, I guess it's already in MultiSegmentPredictionIntervalModel?